docs/components/FileInputView.less

```diff
@@ -0,0 +1,7 @@
+@import (reference) "~less/index";
+
+
+.TextInputView--inputContainer {
```

Comment:
Yea good call - just going to remove this less file, the docs look fine without it.